modules/web/Makefile

```
@@ -0,0 +1,47 @@
+# Web UI Arguments (overridable)
+SYSTEM_BANNER ?= "Local test environment"
+SYSTEM_BANNER_SEVERITY ?= INFO
+AUTO_GENERATE_CERTIFICATES ?= false
+
+# Angular Serve Arguments
+PROXY_CONFIG ?= proxy.conf.json
+SSL_ENABLED ?= false
+
+# List of targets that should be always executed before other targets
+PRE = --ensure-version
+
+MAKEFLAGS += -j2
+
+.PHONY: serve
+serve: $(PRE) --serve-go --serve-angular
+
+.PHONY: serve-https
+serve-https: $(PRE) --serve-go-https --serve-angular-https
+
+.PHONY: build
+build: $(PRE)
+	scripts/build.sh
+
```

```suggestion

.PHONY: ci
ci:
	npm ci

```
Install all necessary NPM tools for building, and validating the version.

Otherwise you end up with an error:
```
node ./scripts/version.mjs
node:internal/errors:465
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'git-describe' imported from /go/src/github.com/kubernetes/dashboard/modules/web/scripts/version.mjs
```